opentelemetry/proto/profiles/v1development/profiles.proto

```diff
@@ -134,6 +117,30 @@ message ProfilesData {
   repeated AttributeUnit attribute_units = 8;
 }
 
+// ProfilesData represents the profiles data that can be stored in persistent storage,
+// OR can be embedded by other protocols that transfer OTLP profiles data but do not
+// implement the OTLP protocol.
+//
+// The main difference between this message and collector protocol is that
+// in this message there will not be any "control" or "metadata" specific to
+// OTLP protocol.
+//
+// When new fields are added into this message, the OTLP request MUST be updated
+// as well.
+message ProfilesData {
+  // An array of ResourceProfiles.
+  // For data coming from an SDK profiler, this array will typically contain one
+  // element. Host-level profilers will usually create one ResourceProfile per
+  // container, as well as one additional ResourceProfile grouping all samples
+  // from non-containerized processes.
+  // Other resource groupings are possible as well and clarified via
+  // Resource.attributes and semantic conventions.
+  repeated ResourceProfiles resource_profiles = 1;
+
+  // An array of ProfilesReferenceTables
+  repeated ProfilesReferenceTables lookup_tables = 2;
```

Comment:
Should this be repeated or we want a single instance of ProfilesReferenceTables? I thought we use one set of reference tables for all ResourceProfiles. Did I misunderstand that?